marshmallow_jsonapi/fields.py

```diff
@@ -7,12 +7,12 @@
 
 from marshmallow.compat import basestring
 
-from marshmallow import ValidationError, class_registry, missing
+from marshmallow import ValidationError, class_registry
 from marshmallow.fields import Field
 # Make core fields importable from marshmallow_jsonapi
 from marshmallow.fields import *  # noqa
 from marshmallow.base import SchemaABC
-from marshmallow.utils import is_collection
+from marshmallow.utils import is_collection, missing as missing_
```

Comment:
Just wondering: why was the import changed?

Reply:
Match import style in [marshmallow/fields.py](https://github.com/marshmallow-code/marshmallow/blob/dev/marshmallow/fields.py#L16).